src/constants.js

```diff
@@ -0,0 +1,3 @@
+export const ADD_BOOK = 'ADD_BOOK';
+export const DELETE_BOOK = 'DELETE_BOOK';
+export const STATUS_CATEGORIES = 'STATUS_CATEGORIES';
```

Comment:
- [ ] The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux) 😉

Reply:
Can you please give me more clues on that